_posts/2017-02-07-how-to-test-a-react-app-using-capybara-webkit.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "How to test a React app using capybara-webkit"
+date: 2017-02-07 13:52:00
+categories: ["rails", "react"]
+author: "mauro-oto"
+---
+
+How to test a React app using capybara-webkit
+
+I have been using the [capybara-webkit](https://github.com/thoughtbot/capybara-webkit)
+gem for a while now since I first tried it out after making the switch from
+Capybara + Selenium.
+
+Using capybara-webkit speeds up my Selenium tests due to its headless nature,
+and it's very straight-forward to use. However, I had some trouble testing a
+Rails based React app.
```

Comment:
Also, maybe you can explain that you're not using web pack and that you're using the react-rails gem.